sim/drivers/infinisim/Spi.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Spi.h"
```

Comment:
absolute paths for includes please

```suggestion
#include "drivers/infinisim/Spi.h"
```